Assets/Scripts/Canvas/PropertiesMenuHandler.cs

```diff
@@ -48,16 +57,29 @@ public void Start()
 
     public void Update()
     {
+      if (_nodeClasses == null)
+      {
+        _graphService = FindObjectOfType<GraphService>();
+        if (_graphService != null)
+        {
+          var classes = FindObjectOfType<GraphService>().Graph.classes;
+          _nodeClasses = new List<NodeClass> { null }.Union(classes).ToList();
```

Comment:
this seems rather... unconventional

Reply:
Yeah it was quick and worked 😄 . This code will be overhauled anyway

Reply:
I think I would like
nodeClassDropdown.AddOptions(new List<>(c_noneClassLabel).Union( _nodeClasses.Select(n => n.name)).ToList())
with name being obligatory

Reply:
Yeah but we still need a null in _nodeClasses that represents no object.
I don't like this approach but we have to keep the optional stuff and nulls are quickest way to do it.

Reply:
ok,, so maybe the Null Object pattern for empty class?

Reply:
I'd prefer Option, I will look into this